lib/icingadb/icingadb-objects.cpp

```diff
@@ -181,6 +177,9 @@ void IcingaDB::UpdateAllConfigObjects()
 
 	upq.ParallelFor(types, [this](const TypePair& type) {
 		String lcType = type.second;
+		ConfigType *ctype = dynamic_cast<ConfigType *>(type.first.get());
+		if (!ctype)
```

Comment:
Side note (not directly related to this PR): the possible values of `type` are hardcoded above. Shouldn't there be a way to ensure that this cast will work? But I haven't been able to figure out the relation of these types in the class hierarchy yet (yay, auto-generated code...)

Reply:
Yes, that's what I thought when moving that code. Didn't remove it because I wasn't entirely sure how those types relate.